### PR TITLE
Attemt to fix #111 by post-processing AWS JSON spec

### LINKED ISSUE #111: Missing AWS::EC2::VPCEndpointService and other discrepancies
Hi, guys!
This is probably more of an FYI and a question than an issue report but wanted to mention.

We ran into an issue with `AWS::EC2::VPCEndpointService` being missing in the API, which turned out to be an AWS problem, we have a support ticket open.
Similarly just bumped into another issue where AWS published resource property as an `VPCEndpointType` where's it has to be `VpcEndpointType`.

Both issues are clearly AWS problems, one of them was known to AWS for a while and they already had an internal ticket ... which brings a question of how to deal with that?

For now I'm just "forking" the project and will attempt to "fix" the downloaded AWS JSON by hand but there's got to be a more intelligent way of "patching" the official but incorrect specifications.

Any good suggestions? Is it worth adding some sort of formal `CloudFormationResourceSpecification.json` patching mechanism to this project?

Thanks!

### PR DESCRIPTION
**Pull request by andreyk0**

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-resolver: nightly-2018-10-01
+resolver: lts-13.7
```
Comment (jdreaver) [inline stack.yaml]: This should probably be its own PR. However, if it compiles I'm happy keeping it.

Comment (andreyk0) [reply stack.yaml]: It compiles but I'll re-submit as a separate PR on Mon, you are right.

**Comment by andreyk0** [comment]
Thanks! I already had a ticket open with them about some of this stuff but AWS never gives an ETA.
All they gave me was "watch our blog" links.
Of course if you have a better luck with them and they fix it ASAP - happy to remove some of these extra complications.
I'll pull out lts change into a separate PR and re-submit on Mon.
Appreciate you taking a look at this!

**Comment by andreyk0** [comment]
I'm going to close this one and re-submit in a bit as 2 separate PRs.
